pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -207,6 +211,10 @@ func (a *Bootstrap) addTemporaryBootkubeFiles(templateData *bootstrapTemplateDat
 			ignition.FileFromString(filepath.Join(podCheckpointerBootstrapDir, name), 0644, data),
 		)
 	}
+	a.Config.Storage.Files = append(
+		a.Config.Storage.Files,
+		ignition.FileFromString(filepath.Join(podCheckpointerBootstrapDir, "pod-checkpointer-kubeconfig.yaml"), 0644, applyTemplateData(content.BootkubeCheckpointerKubeConfig, templateData)),
```

Comment:
Do you know where this comes from?  I would just as soon delete the entire checkpointer and not wait for that pod.

@sjenning same question.

Reply:
Am fine with deleting the checkpointer right away.

Reply:
https://github.com/openshift/installer/pull/591

Reply:
rebased and removed all checkpointer code from here.